### DISCUSSION TITLE
Show Ulauncher on key release

### DISCUSSION THREAD

**Discussion by sematula** [Ulauncher/Ulauncher#912] [Q&A] [answered]
This isn't a problem so much as it's a question ... is there a way to configure ulauncher to appear when the bound key is _released_? I'd like to keep my use of the super key consistent across OSs, which means SUPER is used to launch applications, while SUPER+ arrow keys moves windows around the desktop.

Thanks in advance.

**Comment by gornostal** [comment]
Unfortunately, it's not possible in the current version.

I gave it a thought, and decided I'm not going to add this feature because it requires a lot of changes. 

Currently my goal for Ulauncher is to reduce the amount the new code to the minimum, so I can keep up with bug fixes and support on my own.

**Comment by eugenesvk** [comment]
@gornostal would you kindly reconsider? Currently yours is the only launcher that is working with my `Hyper L` key mapped to the useless `Caps Lock`, however, I'm using `Hyper+X` for a quick launch/switch to the already launched apps, and Ulauncher breaks it :(
This means that I can't really replicate the simple and very convenient Windows experience of having `Caps Lock` launch launcher on a single key press and work as a launcher/switcher itself when pressed as part of a key combo
Would really appreciate it!

**Comment by gornostal** [comment]
@eugenesvk reopening the ticket.

Looks like more and more people would like to have this feature

**Comment by goodwillcoding** [comment]
@eugenesvk ... this feature would move forward with a nice PR. Would you consider doing the work?

**Comment by eugenesvk** [comment]
sorry, @goodwillcoding, but don't know any coding to be able to help :(

**Comment by friday** [comment, marked as answer]
As mentioned earlier it's not possible to support this using the keybinder library
https://lazka.github.io/pgi-docs/#Keybinder-3.0

You would have to set the Ulauncher keyboard combination to some custom solution to handle the evdev events (likely need to run as root or will only work for x11), then handle the key release event externally and trigger `ulauncher-toggle`

https://blog.robertelder.org/detect-keyup-event-linux-terminal/
